### PR TITLE
support configuring misc http headers for Graphite requests

### PR DESCRIPTION
**Pull request by Dieterbe**

### CODE REVIEW

**Comment by captncraig** [comment]
Code looks good to me. What is you use case for this?

**Comment by Dieterbe** [comment]
We use the litmus graphite-kairosdb storage plugin which uses an x-org-id header for authorization

**Comment by kylebrandt** [comment]
Litmus is raintanks upcoming sass monitoring product (for context)

File: docs/configuration.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Every variable is optional, though you should enable at least 1 backend.
   * The items page.
   * The graph page's tag list.
 * graphiteHost: an ip, hostname, ip:port, hostname:port or a URL, defaults to standard http/https ports, defaults to "/render" path.  Any non-zero path (even "/" overrides path)
+* graphiteHeader: a http header to be sent to graphite on each request in 'key:value' format. optional. can be specified multiple times.
```
Comment (mjibson) [inline docs/configuration.md]: I think you must add an exception to the `seen` func to make this specifiable multiple times.

Comment (Dieterbe) [inline docs/configuration.md]: you're right, fixed in https://github.com/bosun-monitor/bosun/pull/1234
thanks for still looking out matt ;)

**Pull request merged**